src/test/kotlin/org/coepi/api/base/ReportsTableTest.kt

```diff
@@ -0,0 +1,13 @@
+package org.coepi.api.base
+
+import com.amazonaws.services.dynamodbv2.datamodeling.*
+
+@DynamoDBTable(tableName = "Report")
+data class ReportsTableTest(
```

Comment:
Why is this needed ? We can simply use the table models that we already have.

Reply:
It requires to modify the existing Report class.